kDrive/UI/Controller/LoginDelegateHandler.swift

```diff
@@ -64,6 +65,10 @@ public final class LoginDelegateHandler: InfomaniakLoginDelegate {
         UserDefaults.shared.legacyIsFirstLaunch = false
         UserDefaults.shared.numberOfConnections = 1
         _ = router.showMainViewController(driveFileManager: driveFileManager, selectedIndex: nil)
+
+        Task {
+            deeplinkService.processDeeplinksPostAuthentication()
+        }
```

Comment:
This looks weird, Task with no await and no MainActor ? Also `processDeeplinksPostAuthentication` already has a Task inside implementation.